NSS-Wacky-Eels/BangazonSite
# Kr fixes

## Link to Ticket


## Description of Proposed Changes
- Created ProductSearch ViewModel
- Edited page user is pushed to when they log out

## Steps to Test

Outline the steps to test
log in and test search bar again
logout and make sure you are pushed to the Bangazon Top Twenty Home page

```
git fetch --all
git checkout <kr-fixes>
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductController
*Layout.cshtml
*Startup.cs

## Mentions @username

@alexfont321
@priyankagargc27
@KaylaReid
@thatmikeparrish
@Akeen11

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
